test/test_cliptok.py

```diff
@@ -1,10 +1,11 @@
 import unittest
 import numpy as np
+import numpy.lib.recfunctions as nlr
 import onnxruntime as _ort
 
 from pathlib import Path
 from onnx import helper, onnx_pb as onnx_proto
-from transformers import CLIPTokenizer
+from transformers import CLIPTokenizerFast
```

Comment:
why switch to the *Fast version?
nonFast version seems to be more formal

Reply:
normal version does not support offset mapping which we need for test validation

Reply:
If there is no official support in Normal version, and if there is no good solution for reserve vocabulary looking up, we may not push for this feature.

Reply:
sounds good, we can keep offset mapping on a case-by-case implementation based on customer demand in that case, like with RobertaTokenizer.